Clear guide for new user installation

anthonyargatoff opened the pull request:
Wrote a simple and clear guide for a new user to start the application. Removed other non-necessary information from the readme (requirements)

Closes #182

Varun2711 reviewed (changes requested):
All the steps are numbered as step 1, could you please number them accordingly, for the python install while it should work fine with python 3.8+ just to be safe could we specify install to be 3.10+ (I think most of us used 3.10) so dependency compatibility won't be an issue.  finally executing the main.py will also start the application, so they won't need to use the cli all the time to run the app.

DyPickard reviewed (approved):
Looks good

anthonyargatoff commented:
> All the steps are numbered as step 1, could you please number them accordingly, for the python install while it should work fine with python 3.8+ just to be safe could we specify install to be 3.10+ (I think most of us used 3.10) so dependency compatibility won't be an issue. finally executing the main.py will also start the application, so they won't need to use the cli all the time to run the app.

In markdown a list is rendered properly even if incorrect numbers are used. It is preferable to label lists like this in the case that we want to add or edit our list. I've updated the version requirement and added that the app can be started with `python main.py`
